src/components/molecules/ArticleListTrashed/index.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div class="article-list">
+    <div v-if="errorMessage" class="article-list__notice--error">
+      <app-text bg-error>{{ errorMessage }}</app-text>
+    </div>
+    <app-heading :level="1">{{ articleTitle }}</app-heading>
+    <app-router-link
+      to="/articles"
+      class="article-list__link"
+      key-color
+      white
+      bg-lightgreen
+      small
+      round
+      hover-opacity
+    >
+      すべての記事一覧へ戻る
+    </app-router-link>
+    <table class="article-list__table">
+      <thead class="article-list__table__head">
+        <tr>
+          <th v-for="item in theads" :key="item">
+            <app-text tag="span" theme-color bold>
+              {{ item }}
+            </app-text>
+          </th>
+        </tr>
+      </thead>
+      <transition-group
+        class="article-list__table__body"
+        name="fade"
+        tag="tbody"
+      >
+        <tr v-for="article in targetArray" :key="article.id">
+          <td>
+            <app-text tag="span" small>
+              {{
+                shortenStrWithEllipsis(article.title)
+              }}
+            </app-text>
+          </td>
+          <td>
+            <app-text tag="span" small>
+              {{
+                shortenStrWithEllipsis(article.content)
+              }}
+            </app-text>
+          </td>
+          <td>
+            <app-text tag="span" small>
+              {{ article.created_at.slice(0, 10) }}
+            </app-text>
+          </td>
+        </tr>
+      </transition-group>
+    </table>
+  </div>
+</template>
+
+<script>
+import { Heading, RouterLink, Text } from '@Components/atoms';
+
+export default {
+  components: {
+    appHeading: Heading,
+    appRouterLink: RouterLink,
+    appText: Text,
+  },
+  props: {
+    targetArray: {
+      type: Array,
+      default: () => [],
+    },
+    borderGray: {
+      type: Boolean,
+      default: false,
+    },
+    title: {
+      type: String,
+      default: '削除済み記事',
+    },
+    errorMessage: {
+      type: String,
+      default: '',
+    },
+  },
+  data() {
+    return {
+      theads: ['タイトル', '本文', '作成日'],
+    };
+  },
```

Comment:
[imo]
ただの定数だと思うのでリアクティブプロパティとして管理する必要なないかなと思いました！
data内はリアクティブプロパティを管理する場所なので、リアクティブである必要がない値はなるべくここにはおきたくないです！

```suggestion
  theads: ['タイトル', '本文', '作成日'],
```
export defaultの中にtheadsを入れてtemplateからは[vm.$opitons](https://v2.ja.vuejs.org/v2/api/#vm-options)を使用することで参照することができます。

Reply:
修正しました！